packages/audio/bluez-alsa/package.mk

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+# Copyright (C) 2022-present Fewtarius
```

Comment:
These aren't my packages, they probably shouldn't have my copyright applied to them.

Reply:
Fair point - fixed.